### DISCUSSION TITLE
Gaming on WSA

### DISCUSSION THREAD

**Discussion by abumettleq** [microsoft/WSA#497] [General]
Does WSA support 1080p + 90fps (or higher) gaming? If not, is there any future plans on adding such feature?

**Comment by zoey29o** [comment]
> Does WSA support 1080p + 90fps (or higher) gaming? If not, is there any future plans on adding such feature?

ah if you have a second graphic card yes if you have IGPU graphics card it doesnt go above 30 FPS regardless resolution y played even o 4k it won't change a thing.

**Reply by abumettleq** [reply]
I have a 1080ti installed on my MB,  I tried running PUBGM on WSA and the resolution the game was showing is 720p!

**Reply by zoey29o** [reply]
well, that is because the setup is similar to a pixel 5 device if you want just for gaming get the google play games for pc it runs on android 14 is supports full Vulkan https://play.google.com/googleplaygames

and if you want to have custom resolution and refresh rates you get the developer edition 
https://developer.android.com/games/playgames/emulator
downside no root solution as we speak, and other apps can only be sideloaded.

**Reply by v8karlo** [reply]
If I understood corectlly it is dead end !? I had a hope it works , but ... ok , bye !

**Reply by Slion** [reply]
Can you run any android app on that Google Play Games emulator?

**Reply by WesleyVanNeck** [reply]
is there a wat to enable the navebar
